src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.ReversedBinaryOperations.tt

```diff
@@ -106,7 +105,7 @@ namespace Microsoft.Data.Analysis
                         }
                     }
                 default:
-                    throw new NotSupportedException();
+                    throw new NotImplementedException();
```

Comment:
This should stay `NotSupportedException`. `NotImplementedException` is for things that we will eventually implement, but haven't gotten a chance yet. `NotSupported` means we don't have plans to support it.